InOut/CMakeLists.txt

```diff
@@ -34,14 +34,24 @@ if(USE_PORTMIDI)
     endif()
 endif()
 
-
 if(USE_JACK)
     find_package(Jack MODULE)
 endif()
 
+if(APPLE)
+    find_path(COREMIDI_INCLUDE_PATH CoreMIDI.h)
```

Comment:
Why are we using find_path/find_libary when the next part we use find_package?

Reply:
Hmm I'm not actually sure. Let me check again why this was done this way

Reply:
I think this section is redundant with the find_package usage below. Could you delete this block (lines 41-48)? Seems to work here on my computer but would like to see this build on GH actions. I think this is the last thing to resolve before merging in.

Reply:
(I went ahead and made the change and pushed to test)